azure-quantum/azure/quantum/qiskit/backends/qci.py

```diff
@@ -0,0 +1,109 @@
+##
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+##
+import warnings
+
+from typing import TYPE_CHECKING, Union, List
+from azure.quantum.version import __version__
+from azure.quantum.qiskit.job import AzureQuantumJob
+
+from .backend import AzureBackend, QIR_BASIS_GATES
+
+from qiskit.providers.models import BackendConfiguration
+from qiskit.providers import Options
+
+if TYPE_CHECKING:
+    from azure.quantum.qiskit import AzureQuantumProvider
+
+import logging
+logger = logging.getLogger(__name__)
+
+__all__ = [
+    "QCISimulatorBackend"
```

Comment:
Adding a comment to ensure this doesn't get lost-
As per Andres, we need to add QCIQPUBackend here